webapp/src/app/components/leaderboard/leaderboard.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="flex flex-col gap-4 py-8">
+  <h1 class="text-3xl font-bold">Artemis Leaderboard</h1>
+  @if (query.isPending()) {
+    <span class="text-muted-foreground">Data is loading...</span>
+  } @else if (query.error()) {
+    <span class="text-destructive">An error has occurred</span>
+  }
+
+  @if (leaderboard(); as leaderboard) {
+    <div class="px-4">
+      <app-table>
+        <thead appTableHeader>
+          <tr appTableRow>
+            <th appTableHead>Contributor</th>
```

Comment:
A column for placement would be good I guess

Reply:
Really like this suggestion! Added this in 7472b20. 
Unfortunately, this meant that I had to switch from `record` to `class` for the `LeaderboardEntry` since the rank is not known on initialization.

Reply:
This is fine for now, in the future we might directly fetch the entries from the database like some kind of time series. Then we could directly fetch the entries from the db where we can use the rank we get through sorting by score